### ISSUE TITLE
Navigation: invisible links should not be keyboard focussable

### ISSUE THREAD

**Issue by FoxyHunter7** [eurofurence/website#30]
[WCAG: Focus Not Obscured (Minimum) (Level AA) ](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html)

On mobile when the navigation is closed the links remain in the tab order.

Although a small issue, if someone would for some reason be on the mobile view on a non-touch device and navigate the site without a mouse they could be navigating navigation items which they cannot see.
